src/components/Comments/WriteComment.js

```diff
@@ -97,6 +112,8 @@ const WriteComment = ({
               style={styles.image}
               source={{
                 uri: profile.myProfile.profile_pic_path
+                  ? profile.myProfile.profile_pic_path
+                  : DEFAULT_PROFILE_PIC_PATH
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good addition of a fallback to `DEFAULT_PROFILE_PIC_PATH` when `profile.myProfile.profile_pic_path` is not available.



<!-- This is an auto-generated comment by OSS CodeRabbit -->